### PR TITLE
Adding Github Actions and small changes on structure of tests

### PR DESCRIPTION
**Pull request by pizzato**
Small changes as making sure rescale is done after new plaintext
Disabling CKKS tests until we are able to fix some rescaling problems
Creating a workflow for github actions

### CODE REVIEW

**Comment by pizzato** [comment]
Adding Badge..

File: .github/workflows/pythonpackage.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```
Comment (CJRChang) [inline .github/workflows/pythonpackage.yml]: Lets use ubuntu-18.04 in case ubuntu-latest changes without warning.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
 # pyHeal
+![PyHEAL Package](https://github.com/pizzato/pyheal/workflows/PyHEAL%20Package/badge.svg)
+
```
Comment (CJRChang) [inline README.md]: Please update badge to
`PyHEAL Package`

**Review by CJRChang — CHANGES_REQUESTED** [review]
A few minor changes required. CKKS can be fixed in a separate request.

**Review by CJRChang — APPROVED** [review]

**Pull request merged**